app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/components/PasswordOptionAlertDialog.kt

```diff
@@ -75,9 +72,10 @@ fun PasswordOptionAlertDialog(
         descriptionRes = R.string.settingsPasswordDescription,
         onDismiss = ::onDismiss,
         onConfirmation = ::onConfirmButtonClicked,
-        shouldEnableConfirmButton = { if (isPasswordActivated) isPasswordValid() else true },
+        shouldEnableConfirmButton = { !isPasswordActivated || isPasswordValid() },
```

Comment:
I feel like we often we have properties called `isEnabled` in google's components. Do you think we could do closer to the usual naming? Maybe something like `isConfirmButtonEnabled`?